guides/common/modules/proc_enabling-dependency-solving-for-a-content-view.adoc

```diff
@@ -1,7 +1,7 @@
-[id="enabling-dependency-solving-per-content-view_{context}"]
-= Enabling Dependency Solving Per Content View
+[id="enabling-dependency-solving-for-a-content-view_{context}"]
+= Enabling Dependency Solving for a Content View
 
-You can enable dependency solving for a Content View or by default for your whole {Project} environment.
+You can enable dependency solving for a Content View or by default for your {Project} environment.
```

Comment:
```suggestion
You can enable dependency solving for a Content View.
```

I would leave off this part of the sentence. It's accurate, but only for the older versions where the setting was still there and did something. Leaving it out makes the sentence correct for all versions.